test/TestCliRolePaths.py

```diff
@@ -8,10 +8,13 @@ class TestCliRolePaths(unittest.TestCase):
     def setUp(self):
         self.local_test_dir = os.path.dirname(os.path.realpath(__file__))
 
-    def run_ansible_lint(self, cwd, role_path, bin=None, env=None):
+    def run_ansible_lint(self, cwd, bin=None, role_path=None, env=None):
         if not bin:
             bin = sys.executable + " -m ansiblelint"
-        command = '{} {}'.format(bin, role_path)
+        if role_path:
+            command = '{} {}'.format(bin, role_path)
+        else:
+            command = '{} -v'.format(bin)
```

Comment:
Would something like this work?
```suggestion
        command = '{bin!s} {role_path or "-v"!s}'.format_map(locals())
```

Reply:
That one would not work and even if it would work it would not add any real benefits to the code.

Ditching an easy to read if for a complex one-liner does not make code easier to understand, especially as it would still hide the conditional. I will try another approach.